Broken husky hooks

I experienced 2 issues with the lint staged:

1. when committing changes to multiple packages, lint-staged with lerna exploded :boom: I had to skip the hooks to commit. I'll post the console output in a follow up comment. This probably requires the concurrency setting with lerna `lerna run lint:staged --concurrency 1`
1. husky v5 is not backwards compatible with v4 which was merged in #640 . See the [husky docs](https://typicode.github.io/husky/#/?id=migrate-from-v4-to-v5). Seems the hooks and lint-staged are now disabled :cry:

Here is the output from the faling lint-staged:

```
$ git commit -m "feat: todos list"
husky > pre-commit (node v12.19.0)
> titus@1.0.0 lint:staged /home/cody/dev/nearform/titus-cody
> lerna run lint:staged
lerna notice cli v3.22.1
lerna info Executing command in 3 packages: "npm run lint:staged"
lerna info run Ran npm script 'lint:staged' in 'titus-db-manager' in 0.4s:
> titus-db-manager@1.0.0 lint:staged /home/cody/dev/nearform/titus-cody/packages/titus-db-manager
> lint-staged
ℹ No staged files match any configured task.
lerna info run Ran npm script 'lint:staged' in 'titus-backend' in 1.5s:
> titus-backend@1.0.0-alpha.1 lint:staged /home/cody/dev/nearform/titus-cody/packages/titus-backend
> lint-staged
[STARTED] Preparing...
[SUCCESS] Preparing...
[STARTED] Running tasks...
[STARTED] Running tasks for *.js
[STARTED] eslint --fix
[SUCCESS] eslint --fix
[STARTED] prettier --write
[SUCCESS] prettier --write
[SUCCESS] Running tasks for *.js
[SUCCESS] Running tasks...
[STARTED] Applying modifications...
[SUCCESS] Applying modifications...
[STARTED] Cleaning up...
[SUCCESS] Cleaning up...
lerna ERR! npm run lint:staged exited 1 in 'titus-frontend'
lerna ERR! npm run lint:staged stdout:
> titus-frontend@1.0.0-alpha.1 lint:staged /home/cody/dev/nearform/titus-cody/packages/titus-frontend
> lint-staged
[STARTED] Preparing...
[SUCCESS] Preparing...
[STARTED] Running tasks...
[STARTED] Running tasks for *.{js,css,md}
[STARTED] eslint --fix
[SUCCESS] eslint --fix
[STARTED] prettier --write
[SUCCESS] prettier --write
[SUCCESS] Running tasks for *.{js,css,md}
[SUCCESS] Running tasks...
[STARTED] Applying modifications...
[SUCCESS] Applying modifications...
[STARTED] Cleaning up...
lerna ERR! npm run lint:staged stderr:
[FAILED] lint-staged automatic backup is missing!
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! titus-frontend@1.0.0-alpha.1 lint:staged: `lint-staged`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the titus-frontend@1.0.0-alpha.1 lint:staged script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
npm ERR! A complete log of this run can be found in:
npm ERR!     /home/cody/.npm/_logs/2020-12-09T16_28_27_754Z-debug.log
lerna ERR! npm run lint:staged exited 1 in 'titus-frontend'
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! titus@1.0.0 lint:staged: `lerna run lint:staged`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the titus@1.0.0 lint:staged script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
npm ERR! A complete log of this run can be found in:
npm ERR!     /home/cody/.npm/_logs/2020-12-09T16_28_27_780Z-debug.log
husky > pre-commit hook failed (add --no-verify to bypass)
```

--- a/package.json
+++ b/package.json
@@ -18,7 +18,7 @@
     "db:up": "lerna run --stream db:up",
     "lint:all": "lerna run --parallel lint",
     "lint:fix": "lerna run --parallel lint:fix",
-    "lint:staged": "lerna run lint:staged",
+    "lint:staged": "lerna run lint:staged --concurrency 1",
     "postinstall": "lerna bootstrap",
     "start:all": "lerna run --parallel start",
     "start:docs": "docsify serve ./docs --port 4000",
@@ -30,7 +30,7 @@
   },
   "devDependencies": {
     "docsify-cli": "^4.4.2",
-    "husky": "^5.0.6",
+    "husky": "^4.3.6",
     "jest": "^26.6.3",
     "lerna": "^3.22.1"
   },

--- a/renovate.json
+++ b/renovate.json
@@ -4,6 +4,8 @@
   "labels": ["renovate"],
   "automerge": true,
   "automergeType": "branch",
+  "prCreation": "not-pending",
+  "stabilityDays": 7,
   "dependencyDashboard": true,
   "packageRules": [
     {

